### PR TITLE
Enable to ignore manifests' fields from kubernetes' drift detection

### PR DESCRIPTION
**Pull request by funera1**
**What this PR does / why we need it**:
In the previous implementation, panic occurred when ddCfg was nil. So, this problem is solved.

**Which issue(s) this PR fixes**:

Fixes #4202 #4241

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/driftdetector/kubernetes/detector.go [hunk 1]
```diff
@@ -188,13 +188,24 @@ func (d *detector) checkApplication(ctx context.Context, app *model.Application,
 	liveManifests = filterIgnoringManifests(liveManifests)
 	d.logger.Debug(fmt.Sprintf("application %s has %d live manifests", app.Id, len(liveManifests)))
 
+	ddCfg, err := d.getDriftDetectionConfig(repo.GetPath(), app)
+	if err != nil {
+		return err
+	}
+
+	ignoreFields := []string{}
```
Comment (knanao) [inline pkg/app/piped/driftdetector/kubernetes/detector.go:196]:
```suggestion
	ignoreFields := make([]string, 0)
```

**Review by knanao — COMMENTED** [review]
Could you use rebase to make the commits list clear?

**Comment by funera1** [comment]
> Could you use rebase to make the commits list clear?

OK. I'll try it.

**Comment by funera1** [comment]
@knanao 
Please review this PR when you have time.

**Review by knanao — APPROVED** [review]
Here you go!

**Comment by funera1** [comment]
#4190

**Pull request merged**